### PR TITLE
Update support-center-onetrace.md

### PR DESCRIPTION
**Pull request by codaamok**
Removed lurking quotation mark
Decided to delete rather than match at the end of the string to follow suit with the previous path given for the msi

### Summarize the change in the pull request title

Describe your change, specifically *why* you think it's needed.

Formatting, nothing crazy.

### CODE REVIEW

**Comment by PRMerger8** [comment]
@codaamok : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: sccm/core/support/support-center-onetrace.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ OneTrace works with many types of log files, such as:
 
 OneTrace installs with Support Center. Find the Support Center installer on the site server at the following path: `cd.latest\SMSSETUP\Tools\SupportCenter\SupportCenterInstaller.msi`.
 
-By default, the OneTrace application is installed at `"C:\Program Files (x86)\Configuration Manager Support Center\CMPowerLogViewer.exe`.
+By default, the OneTrace application is installed at `C:\Program Files (x86)\Configuration Manager Support Center\CMPowerLogViewer.exe"`.
```
Comment (aczechowski) [inline sccm/core/support/support-center-onetrace.md]: It looks like the leading quotation mark was removed, but one was added at the end of the string.

```suggestion
By default, the OneTrace application is installed at `C:\Program Files (x86)\Configuration Manager Support Center\CMPowerLogViewer.exe`.
```

You can either simply accept my suggestion, or in the **Files changed** section of the pull request, select the ellipsis (`...`) in the upper right, and select **Edit file**. Make the change into the *same branch* to automatically update this pull request.

**Comment by codaamok** [comment]
LOL

I removed it from one end, and added it at the other end. What a doofus.

**Comment by PRMerger20** [comment]
@codaamok : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by aczechowski** [comment]
Looks good 😄 
#sign-off 
*Once merged*, this should be live with the next daily publishing process. Right now that *could* be today (Friday, 27 December) around 11:00 Pacific time. (If not today, then should be Monday.) No further action needed from you.

**Pull request merged**